### PR TITLE
Adding Path Of Titans Egg

### PR DESCRIPTION
**Pull request by Hendalf-DEV**
# Description

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

* [ ] You verify that the start command applied does not use a shell script
  * [x] If some script is needed then it is part of a current yolk or a PR to add one
* [ ] The egg was exported from the panel

## New egg Submissions

1. [x] Does your submission pass tests (server is connectable)?
2. [ ] Does your egg use a custom docker image?
    * [ ] Have you tried to use a generic image?
    * [ ] Did you PR the necessary changes to make it work?
3. [x] Have you added the egg to the main README.md and any other README files in subdirectories of the egg (e.g /game_eggs) according to the alphabetical order?
4. [x] Have you added a unique README.md for the egg you are adding according to the alphabetical order?
5. [ ] You verify that the start command applied does not use a shell script
    * [x] If some script is needed then it is part of a current yolk or a PR to add one
6. [ ] The egg was exported from the panel

### CODE REVIEW

File: path_of_titans/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Path Of Titans
+
+<!--Please remove these comments and irelevent parts for the server egg your adding before summiting a PR request-->
+
+## From their [Github](https://github.com/Hendalf-DEV/eggs)
```
Comment (QuintenQVD0) [inline path_of_titans/README.md:5]: Wy does this link to you? This is not ment for that. It is meant to link to the server repo if their is one.

File: path_of_titans/egg-path-of-titans (3).json [hunk 2]
```diff
@@ -0,0 +1,112 @@
+{
+    "_comment": "DO NOT EDIT IF YOU DO NOT KNOW WHAT YOU ARE DOING!",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-04-26T06:39:41+00:00",
+    "name": "Path Of Titans",
+    "author": "danielivanvladimir.revin@gmail.com",
+    "description": null,
```
Comment (QuintenQVD0) [inline path_of_titans/egg-path-of-titans (3).json:10]: Maybe add a description?

File: path_of_titans/egg-path-of-titans (3).json [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            "rules": "required|string|max:1500",
+            "field_type": "text"
+        },
+        {
+            "name": "Query port",
+            "description": "",
+            "env_variable": "QUERY_PORT",
+            "default_value": "7778",
+            "user_viewable": true,
+            "user_editable": true,
```
Comment (QuintenQVD0) [inline path_of_titans/egg-path-of-titans (3).json:107]: A port can never be user editable.

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

**Comment by QuintenQVD0** [comment]
Also fix the (3) in the filename please.

File: path_of_titans/README.md (hunk 1 above)
Comment (Hendalf-DEV) [reply path_of_titans/README.md]: Removed.

File: path_of_titans/egg-path-of-titans (3).json (hunk 2 above)
Comment (Hendalf-DEV) [reply path_of_titans/egg-path-of-titans (3).json]: Added.

File: path_of_titans/egg-path-of-titans (3).json (hunk 3 above)
Comment (Hendalf-DEV) [reply path_of_titans/egg-path-of-titans (3).json]: Changed to only viewable

**Comment by Hendalf-DEV** [comment]
Required changes are made.

**Comment by QuintenQVD0** [comment]
@Hendalf-DEV If you can mention in the readme how to get a token then this can be merged.

**Comment by Hendalf-DEV** [comment]
> @Hendalf-DEV If you can mention in the readme how to get a token then this can be merged.

Ready
https://github.com/pelican-eggs/games/commit/0c69469a54a0c6146aad84575cf313134093df5f

**Pull request merged**